frontend/css/playlists.less

```diff
@@ -277,3 +278,25 @@
     box-shadow: 0 0 0 2px rgba(53, 48, 112, 0.5);
   }
 }
+
+.wikipedia-extract {
+  .playlist-description {
+    overflow: hidden;
+    position: relative;
+    line-height: 1.2em;
+    max-height: @playlistDescriptionLines * 1.2em;
+    text-align: justify;
+    padding-right: 2em;
+    display: -webkit-box;
+    line-clamp: @playlistDescriptionLines;
+    margin-top: 0.5rem;
+    -webkit-line-clamp: @playlistDescriptionLines;
+    -webkit-box-orient: vertical;
+  }
+
+  .show-more-button {
```

Comment:
I'm not sure we need custom style here. Instead you should be able to use the classes `btn btn-link pull-right` on the button element like we do for the artist page wikipedia extract